test/tests/functional/pbs_soft_walltime.py

```diff
@@ -852,46 +831,64 @@ def test_walltime_default(self):
         self.server.expect(JOB, {'job_state': 'R'}, id=jid1)
         self.server.deljob(jid1, wait=True)
 
-        try:
-            J = Job(TEST_USER, attrs={'Resource_List.walltime': 10})
-            jid1 = self.server.submit(J)
-        except PtlSubmitError as e:
-            self.assertTrue("illegal resource" in e.msg[0])
+        # following piece is commented due to PP-1058
+        # try:
+        #    J = Job(TEST_USER, attrs={'Resource_List.walltime': 10})
+        #    jid1 = self.server.submit(J)
+        # except PtlSubmitError as e:
+        #    self.assertTrue("illegal attribute or resource value" in e.msg[0])
+        # self.assertEqual(jid1, None)
 
-    def test_soft_suspension(self):
+    def test_soft_held(self):
         """
-        Test that if job is suspended soft_walltime will not get extended
+        Test that if job is held soft_walltime will not get extended
         """
-
-        J = Job(TEST_USER, {'Resource_List.walltime': 300})
+        J = Job(TEST_USER, attrs={'Resource_List.walltime': '100'})
         jid = self.server.submit(J)
         self.server.alterjob(jid, {'Resource_List.soft_walltime': 7})
         self.server.expect(JOB, {'job_state': 'R'}, id=jid)
 
-        self.server.sigjob(jid, signal='suspend')
-        self.server.expect(JOB, {'job_state': 'S'}, id=jid)
+        now = int(time.time())
 
+        # Sleep so long that soft_walltime get extended once
         time.sleep(7)
-        self.server.expect(JOB, {'Resource_List.soft_walltime': 7}, id=jid)
+        self.server.manager(MGR_CMD_SET, SERVER, {'scheduling': 'True'})
+        self.server.expect(JOB, {'estimated.soft_walltime':
+                                 (MATCH_RE, '00:00:14|14')}, id=jid)
 
-        self.server.sigjob(jid, signal='resume')
-        self.server.expect(JOB, {'job_state': 'R'}, id=jid)
+        self.server.holdjob(jid, 'u')
+        self.server.rerunjob(jid)
+        self.server.expect(JOB, {'job_state': 'H'}, id=jid)
 
-        time.sleep(5)
-        self.server.manager(MGR_CMD_SET, SERVER, {'scheduling': 'True'})
+        # sleep so long to verify that soft_walltime don't change
+        # while job is held
+        time.sleep(10)
```

Comment:
Consider turning this comment into a log message.  It will explain to the person running the test why the test is sleeping.